### ISSUE TITLE
Lidar

### ISSUE THREAD

**Issue by nathankota** [agrc/planning-queue#113]
state procurement Multiple Award contract
acquisition
distribution
maintenance

**Comment by rkelson** [comment]
Had the kick-off meeting today with Jamie Huff (Project Manager) and the vendor for the project Aero-Graphics. I will be providing support throughout the project as the Technical Specialist on the project. 

Here is a graphic of the proposed project timeline:

![Image](https://github.com/agrc/planning-queue/assets/6200941/26080937-887e-45d0-bc4c-f308a69a4fbf)

**Comment by rkelson** [comment]
In the NSGIC 3DEP meeting today it sounds like Contributed Data, as this project would be, are at a complete stand still at NGTOC for QA. NGTOC does not have the capacity they need and GPSC projects are their priority. We might need to find someone other than the USGS NGTOC to provide QA on the project deliverables. I've emailed Brian Hadley with the USGS for guidance on this issue.

**Comment by rkelson** [comment]
Had a call with Brian and he has some ideas. He is going to dream on it and give me a call next week.

**Comment by rkelson** [comment]
Waiting to hear back from Brian Hadley (USGS),  Cynthia Ritmiller (IUSGS), or Paul Rooney (FEMA). They are all aware of the issue with contributed data and NGTOC reviews and I am hoping they can figure out a resolution before we start receiving project deliverables.

**Comment by rkelson** [comment]
Brian Hadley (USGS) continues to pursue an avenue for our contributed data to get into the que for USGS Data Validation (QA)

Received a Pilot data delivery for the project. Reviewing it now and providing feedback to the vendor.

**Comment by rkelson** [comment]
2/7/24 - Received and reviewed pilot data for the project. Provided feedback to AGI on the tile sizes and naming convention. Other than those issues things are looking good. They still plan on a mid summer delivery for the entire project. 

Still no resolution with the USGS on the Data Validation (QA) piece.

**Comment by rkelson** [comment]
March 8 project update has pushed the initial data delivery up a month to 5/31/24. This is a month earlier than originally planned.

Still no resolution with the USGS on the Data Validation (QA) piece for Contribute Data.

**Comment by rkelson** [comment]
Brian Hadley with USGS stated 3/28/24:
"Yes, we can help with the validation of the Salt Lake Valley data.
Okay. Sounds good. Let’s check in again as we get closer to the data delivery date."

**Comment by rkelson** [comment]
5/7/24 Met with our new USGS liaison Cynthia Ritmiller and she said to work with her, not Brian, for the USGS Data validation QA piece.

As of today I am waiting to hear from AGI about the first project delivery which should be available any day now.

**Comment by rkelson** [comment]
Just received this from AGI and the initial delivery for the project is postponed until June 30, 2024,
`"Our LiDAR department has detected some issues that are going to require us to deliver at the original end of June due date rather than at the end of May.  I will have a formal PDF update for you early next month. Please let me know if you have any specific questions in the meantime."
`

**Comment by rkelson** [comment]
latest vendor project report states the new delivery date is 6/28/2024

**Comment by rkelson** [comment]
Obtained the initial project deliverables for review and USGS QA 7/1/2024. A copy still needs to be sent by AGI to the USGS (likely 7/3/2024) once the USGS determines who exactly to address the delivery to.

**Comment by rkelson** [comment]
I found a couple incomplete DSM tiles and AGI sent me corrected files and an explanation.

A hard drive of the complete project deliverables sent to USGS 7/3/2024 for QA round 1.

**Comment by rkelson** [comment]
Salt Lake City has requested a copy of all preliminary project deliverables. Kevin Bell is doing some test modeling with tree canopies and structures. After getting the OK to distribute the data from Jamie Huff I am in the process of making a copy to give them.

**Comment by rkelson** [comment]
In addition to utilizing the new lidar for tree canopy modeling, Kevin Bell hopes to create new building footprints for Salt Lake County. I mentioned to him Jamie is interested in figuring out the Lowest Adjacent Grade (LAG) for these building footprints. I've provided Kevin more information on the LAG hoping he will develop a process to derive this information.

**Comment by steveoh** [comment]
🔔 @ZachBeck in case you want to be aware of this to consider merging with our statewide copy of Microsoft's building footprints.

**Comment by ZachBeck** [comment]
I'd love to update the building footprints as well as know how he's creating his.

**Comment by rkelson** [comment]
Received from Cynthia @USGS 8/14/24:

```
The data was received in mid-July and has been uploaded into the project que. I reached out to the Elevation Operations team for an estimation on timing.

The project is in the next round of project which means validation should start within the next couple weeks. I should receive a notification when the validation is started and can give the group a heads up.  

With the project only having one work unit we hope the validation is completed quickly. I will keep you updated as I hear info on this project.
```

**Comment by rkelson** [comment]
requests to the USGS on the status of the Data Validation/QA of the project have fallen on deaf ears so no update to report.

**Comment by rkelson** [comment]
USGS responded to my recent request for an update, after 10 days to think about it, with, 'I reached out to the Data validation team but haven’t heard anything yet. I will reach out as soon as I heard something.

Sorry for the delays.'

**Comment by rkelson** [comment]
4/7/25 - Received the initial USGS Data Validation report for the project. There are numerous errors AGI needs to fix. Jamie Huff has granted AGI's request for 30 days (not 21 days) to fix and redeliver the project to the USGS for another round of data validation.

**Comment by rkelson** [comment]
5/6/25 - Aero-Graphics sent corrections to the USGS

**Comment by rkelson** [comment]
6/6/2025 - -Received email from USGS requesting more corrections for the vendor

**Comment by rkelson** [comment]
The final delivery date in the contract, between UDEM and AGI for the project, has been extended to December 31, 2025 based off the continuing USGS QA calls.

**Comment by rkelson** [comment]
7/3/2025 AGI sent another round of corrections to the USGS. Hopefully this is the last round of corrections and the USGS will accept the project.

**Comment by rkelson** [comment]
8/4/2025 USGS says the LAS tiles that were redelivered now have a problems with the header information. AGI now needs to make another delivery to the USGS.

**Comment by rkelson** [comment]
Final corrections were provided to the USGS today. Hopefully they are accepted quickly and we can finalize the project.

**Comment by rkelson** [comment]
I picked up a hard drive today of the final project deliverables and plan to start work on [718](https://github.com/agrc/planning-queue/issues/718)

**Comment by rkelson** [comment]
Still waiting on the final QA report from the USGS

**Comment by rkelson** [comment]
12/2/2025 - Received notification from the USGS that the project is being published and has been accepted.
